docs/yocto-project/glossary.md

```diff
@@ -3,12 +3,18 @@
 #### UPDATEHUB_PRODUCT_UID
  Identifies the product id in use. This is used by rollouts.
 
-#### UPDATEHUB_UHUPKG_PUBLIC_KEY / UPDATEHUB_UHUPKG_PRIVATE_KEY   
+#### UPDATEHUB_UHUPKG_PUBLIC_KEY
 
-  The variables are required to point to the keys which are used to validate and
-sign the update package.  
+The variable are required to point to the public key which are used to validate and
+sign the update package.
+The public key works together with the private key that is described below.
```

Comment:
here it should mention the private key variable here, so it has an explicit connection.

Reply:
pls see modification